docs/build-intercept.rst

```diff
@@ -20,21 +22,21 @@ can find the libraries and headers it needs:
 
 .. code-block:: Bash
 
-	git clone https://github.com/spack/spack
-	. spack/share/spack/setup-env.sh
-	spack install leveldb
-	spack install gotcha
-	spack install environment-modules
-	spack load environment-modules
-	source <(spack module loads gotcha leveldb)
+	$ git clone https://github.com/spack/spack
+	$ . spack/share/spack/setup-env.sh
+	$ spack install leveldb
+	$ spack install gotcha
+	$ spack install environment-modules
+	$ spack load environment-modules
+	$ source <(spack module loads gotcha leveldb)
```

Comment:
While you're here, the above paragraph should begin "*To* install leveldb ..."

Reply:
@nedbass thanks, this is updated now.